code/modules/clothing/clothing.dm

```diff
@@ -229,6 +229,7 @@ BLIND     // can't see anything
 	var/sensor_mode = 0	/* 1 = Report living/dead, 2 = Report detailed damages, 3 = Report location */
 	var/can_adjust = 1
 	var/adjusted = 0
+	var/modesty = 0 // 0 = exposes chest and arms, 1 = exposes arms only

```

Comment:
Defines isn't right here. we don't use defines for bool vars.

the var name should be changed to altCoversChest alt_covers_chest